src/mpyl/utilities/repo/__init__.py

```diff
@@ -133,6 +134,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     @staticmethod
     def from_clone(config: RepoConfig, repo_path: Path):
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
Why do we need to set this here?

Reply:
I think this might be a leftover from Pedro when he was debugging, will remove it